css/style.css

```diff
@@ -0,0 +1,63 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: url('https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg');
+  padding: 50px 0;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#title,
+#description {
+  padding-bottom: 15px;
+  text-align: center;
+  color: rgb(27, 13, 59);
+}
```

Comment:
Change required:
Your texts in the heading section (title and description ids) don't contrast enough with the background image. It's suggested to change the color in order to improve readability. 👍